_rules/video-only-description-track-ac7dc6.md

```diff
@@ -62,8 +62,8 @@ A video only element with a track element that contains incorrect descriptions.
 
 ```html
 <video controls>
-	<source src="../test-assets/rabbit-video/silent.mp4" type="video/mp4" />
-	<source src="../test-assets/rabbit-video/silent.webm" type="video/webm" />
+	<source src="../test-assets/rabbit-video/silent.mp4" type="video/mp4">
+	<source src="../test-assets/rabbit-video/silent.webm" type="video/webm">
 	<track kind="descriptions" src="rabbit-video-incorrect-descriptions.vtt" />
```

Comment:
`<track>` is also a void element, so it shouldn't have the end slash, similar to `<source>`